Amplify/Core/Support/Task+Seconds.swift

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+extension Task where Success == Never, Failure == Never {
+    static func sleep(seconds: Double) async throws {
+        let nanoseconds = UInt64(seconds * Double(NSEC_PER_SEC))
+        try await Task.sleep(nanoseconds: nanoseconds)
```

Comment:
```suggestion
        try await Task.sleep(nanoseconds: NSEC_PER_SEC * seconds)
```
Can be simplified. :-)

Reply:
Discussed off GH. The `seconds` are `Double` so `UInt64` is required and nesting the work in that parameter quickly becomes less readable.